libs/blocks/caas/utils.js

```diff
@@ -357,7 +371,11 @@ export const getConfig = async (state, strs = {}) => {
       trackImpressions: state.analyticsTrackImpression || '',
       collectionIdentifier: state.analyticsCollectionName,
     },
-    target: { enabled: state.targetEnabled || '' },
+    target: {
+      enabled: state.targetEnabled || '',
+      lastViewedSession: state.lastViewedSession || '',
+    },
+    customCard: ['card', `return \`${state.customCard}\``],
```

Comment:
Just verifying as this seems strange: The 2nd value should be the string "return \`stuff\`"?

Reply:
Yes, where `html stuff` is a either a string or string template.